homeassistant/components/filesize/__init__.py

```diff
@@ -24,6 +25,14 @@ def _check_path(hass: HomeAssistant, path: str) -> None:
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up from a config entry."""
     await hass.async_add_executor_job(_check_path, hass, entry.data[CONF_FILE_PATH])
+
+    path = entry.data[CONF_FILE_PATH]
+    get_path = await hass.async_add_executor_job(pathlib.Path, path)
+    fullpath = str(await hass.async_add_executor_job(get_path.absolute))
```

Comment:
This could be combined into a single executor job

Reply:
I'll combine it with the call above as we don't need to have it separately